lib/main.dart

```diff
@@ -114,42 +103,44 @@ class _MyHomePageState extends State<MyHomePage> {
-            children: <Widget>[
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
-                  child: Icon(Icons.photo_album_outlined),
-                  style: ButtonStyle(
-                    shape: MaterialStateProperty.all(CircleBorder()),
-                    padding: MaterialStateProperty.all(EdgeInsets.all(10)),
-                    backgroundColor: MaterialStateProperty.all(Colors.cyan),
-                    overlayColor: MaterialStateProperty.resolveWith<Color?>((states) {
-                      if (states.contains(MaterialState.pressed)) return Colors.red;
-                    }),
+    return showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            title: const Center(child: Text('Make a Choice...')),
+            content: SingleChildScrollView(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: <Widget>[
+                  ElevatedButton(
+                    onPressed: () => {}, //pickImageFromGallery(),
+                    child: Icon(Icons.photo_album_outlined),
+                    style: ElevatedButton.styleFrom(
+                      shape: CircleBorder(),
+                      padding: EdgeInsets.all(10),
+                      primary: Colors.cyan,
+                    ),
                   ),
-              ),
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
-                  child: Icon(Icons.camera_alt_outlined),
-                  style: ButtonStyle(
-                    shape: MaterialStateProperty.all(CircleBorder()),
-                    padding: MaterialStateProperty.all(EdgeInsets.all(10)),
-                    backgroundColor: MaterialStateProperty.all(Colors.cyan),
-                    overlayColor: MaterialStateProperty.resolveWith<Color?>((states) {
-                      if (states.contains(MaterialState.pressed)) return Colors.red;
-                    }),
+                  ElevatedButton(
+                    // change to better version in top
+                    onPressed: () => {}, //pickImageFromGallery(),
+                    child: Icon(Icons.camera_alt_outlined),
+                    style: ButtonStyle(
+                      shape: MaterialStateProperty.all(CircleBorder()),
+                      padding: MaterialStateProperty.all(EdgeInsets.all(10)),
+                      backgroundColor: MaterialStateProperty.all(Colors.cyan),
+                      overlayColor:
+                          MaterialStateProperty.resolveWith<Color?>((states) {
+                        if (states.contains(MaterialState.pressed))
+                          return Colors.red;
+                      }),
+                    ),
                   ),
+                ],
               ),
-            ],
-          ),
-        ),
-      );
-    });
+            ),
+          );
+        });
```

Comment:
missing trailing comma. Try to use command-line commands:
`flutter analyze .` and `flutter format .` in the main directory of your project (which is higher than lib folder). This will show you all styling problems and eventually will reformat your code after you'll fix all the problems. Please, be patient with commas